client.go

```diff
@@ -372,6 +372,52 @@ func (cli *Client) Versions() (resp *RespVersions, err error) {
 	return
 }
 
+// PublicRooms returns the list of public rooms on target server. Does not require Auth. See http://matrix.org/speculator/spec/HEAD/client_server/unstable.html#get-matrix-client-unstable-publicrooms
```

Comment:
```suggestion
// PublicRooms returns the list of public rooms on the target server.
// Does not require Auth. See https://matrix.org/docs/spec/client_server/r0.6.0#get-matrix-client-r0-publicrooms
```

Except it can require auth these days, so this may need some reworking.

https://github.com/matrix-org/synapse/blob/0495097a7f6c6f5230972e0a5f32c0c9c42ef61b/docs/sample_config.yaml#L68